chrome/skin/style.css

```diff
@@ -166,15 +181,25 @@ popupset#mainPopupSet panel#PopupAutoCompleteRichResult richlistbox {
 #universal-search-recommendation.highlight #universal-search-recommendation-url {
   color: HighlightText;
 }
-#universal-search-recommendation.highlight #universal-search-recommendation-label {
+#universal-search-recommendation.highlight #universal-search-recommendation-label,
+#universal-search-recommendation.highlight #universal-search-recommendation-details,
+#universal-search-recommendation.highlight #universal-search-recommendation-description {
   color: HighlightText;
   opacity: 0.67;
 }
 #universal-search-recommendation.highlight #universal-search-recommendation-image {
   background-color: #FFF;
   box-shadow: 0;
 }
-#universal-search-recommendation.highlight #universal-search-recommendation-logo,
 #universal-search-recommendation.highlight #universal-search-recommendation-keyimage {
   z-index: 1;
 }
+
+#universal-search-recommendation-content {
+  -moz-box-align: stretch;
+}
+
+#universal-search-recommendation-top-content {
+  -moz-box-flex: 1;
+  -moz-box-pack: justify;
```

Comment:
this rule is a duplicate of the rule above. let's merge this into the one above for `#top-content`.